src/tests/ftest/util/osa_utils.py

```diff
@@ -163,6 +177,36 @@ def verify_single_object(self):
         self.obj.close()
         self.container.close()
 
+    def run_ior_thread(self, action, oclass, api, test):
+        """Start the IOR thread for either writing or
+        reading data to/from a container.
+        Args:
+            action (str): Start the IOR thread with Read or
+                          Write
+            oclass (str): IOR object class
+            API (str): IOR API
```

Comment:
```suggestion
            api (str): IOR API
```

Reply:
Sure. I will fix it on the next PR... I have a lot of laundry list to change on osa_utils.py... I willl keep this in mind.